ably/internal/ablyutil/msgpack_test.go

```diff
@@ -31,3 +32,28 @@ func TestMsgpack(t *testing.T) {
 		}
 	})
 }
+
+func Test_msgpackJson(t *testing.T) {
```

Comment:
```suggestion
func TestMsgpackJson(t *testing.T) {
```

(to be consistent with other test names)